ivy/functional/ivy/nest.py

```diff
@@ -1015,6 +1015,49 @@ def nested_map(
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
 
+    Examples
+    --------
+    With :code:`ivy.Array` inputs:
+
+    >>> def special_square(nested : float) -> float : return np.square(x)
+    >>> nested = ivy.Array([[1,2,3],[4,5]])
+    >>> results = ivy.nested_map(x = nested,
```

Comment:
`x` is a positional argument in the function so it should not be passed as `ivy.nested_map(x = nested, fn = special_square )` but just `ivy.nested_map(nested, fn = special_square )`

Reply:
Removed the "x=" part